CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/templates/RecordStoreQueryPluginNGTest.java

```diff
@@ -65,112 +72,189 @@ public void tearDownMethod() throws Exception {
      */
     @Test
     public void testGetResult() {
-        final RecordStoreQueryPlugin instance = new RecordStoreQueryPluginMockImpl();
-        final GraphReadMethods graph = new StoreGraph();
-        final PluginInteraction interaction = null;
-        final PluginParameters parameters = null;
-        try {
-            instance.read(graph, interaction, parameters);
-            instance.query(interaction, parameters);
-        } catch (InterruptedException | PluginException ex) {
-            fail(ex.getLocalizedMessage());
-        }
-        final RecordStore result = instance.getResult();
-        assertEquals(result.size(), 2);
+	System.out.println("getResult");
+
+	final RecordStoreQueryPlugin instance = new RecordStoreQueryPluginMockImpl();
+	final GraphReadMethods graph = new StoreGraph();
+	final PluginInteraction interaction = null;
+	final PluginParameters parameters = null;
+	try {
+	    instance.read(graph, interaction, parameters);
+	    instance.query(interaction, parameters);
+	} catch (InterruptedException | PluginException ex) {
+	    fail(ex.getLocalizedMessage());
+	}
```

Comment:
I think throwing the exception is preferred here

Reply:
Agreed and updated.